frontend/src/pages/Testbed.tsx

```diff
@@ -61,14 +63,81 @@ const logChange = (...args: unknown[]) => {
 
 /** test and example usage of formatting, elements, components, etc. */
 const TestbedPage = () => {
+  /** generate fake node data */
+  const nodes = Array(200)
+    .fill(null)
+    .map(() => ({
+      id: uniqueId(),
+      label: sample([
+        "Lbl.",
+        "Label",
+        "Long Label",
+        "Really Long Label",
+        undefined,
+      ]),
+      type: sample([
+        "gene",
+        "disease",
+        "compound",
+        "anatomy",
+        "phenotype",
+        undefined,
+      ]),
+      strength: sample([0, 0.1, 0.02, 0.003, 0.0004, 0.00005, undefined]),
+      extra: sample(["cat", "dog", "bird"]),
+    }));
+  const ids = nodes.map((node) => node.id);
+  /** generate fake edge data */
+  const edges = Array(500)
+    .fill(null)
+    .map(() => ({
+      id: uniqueId(),
+      label: sample([
+        "Lbl.",
+        "Label",
+        "Long Label",
+        "Really Long Label",
+        undefined,
+      ]),
+      source: sample(ids)!,
+      target: sample(ids)!,
+      type: sample([
+        "causes",
```

Comment:
Similarly, when should these be defined? Accordingly 'define attributes' can be an open issue that reminds us to get back to these later.

Reply:
I think you're asking specifically about the `type` field here?

This is just an arbitrary category that the network component uses to assign different colors to different nodes/edges. It doesn't care what the type is, or have any context for what it means; I've tried to separate the concerns there. 

I was assuming there'd be some use in this, e.g. assigning different species. It could also just as easily be all one color, or random colors. Also if for some reason in the future you wanted to make sure a certain that a specific color was mapped to a specific type, we could easily change it to accept a color mapping instead of the component doing it automatically.

Related, in an above comment you mentioned having different shapes as well. Cytoscape supports that, so we could have a second "type". However I was hoping to reserve different shapes for going along with the colors as a color-blind-accessible option, like so:

![image](https://github.com/user-attachments/assets/9e52eaaa-9492-478f-a54c-f10878ebf15c)